src/geo/lng_lat_bounds.js

```diff
@@ -209,6 +209,25 @@ class LngLatBounds {
         return !(this._sw && this._ne);
     }
 
+    /**
+    * Check if the point is within the bounding box.
+    *
+    * @returns {boolean} True if the point is within the bounding box.
+    */
+    contains(lnglat: LngLatLike) {
+        lnglat = lnglat instanceof LngLat ? new LngLat(lnglat.lng, lnglat.lat) : LngLat.convert(lnglat);
```

Comment:
`LngLat` case is [already covered](https://github.com/mapbox/mapbox-gl-js/blob/master/src/geo/lng_lat.js#L108-L110) in `convert`.

```suggestion
        lnglat = LngLat.convert(lnglat);
```